docs/pe/user-guide/ui/users.md

```diff
@@ -4,8 +4,155 @@ assignees:
+    5:
+        image: /images/user-guide/ui/users/pe/user-edit-5-pe.png
+        title: 'Settings tab is responsible for general permissions and interface of the columns.'
+    6:
+        image: /images/user-guide/ui/users/pe/user-edit-6-pe.png
+        title: 'Actions tab allows navigating to specific dashboard or add a custom action (To learn more about the Actions, see the link below).'
+
+user-delete-pe:
+    0:
+        image: /images/user-guide/ui/users/pe/user-delete-pe.png
+        title: 'To delete a user group, click the trash can icon in the end of its row and confirm deletion.'
+
+user-delete-1-pe:
+    0:
+        image: /images/user-guide/ui/users/pe/user-delete-1-pe.png
+        title: 'To delete a user, click the trash can icon in the end of its row and confirm deletion.'
 
 ---
 
-{% assign docsPrefix = "pe/" %}
-{% include docs/user-guide/ui/users.md %}
\ No newline at end of file
+* TOC
+{:toc}
+
+## Introduction
+
+User is an entity that can log in to the ThingsBoard web interface, execute REST API calls, access devices, assets and other entities if they have permissions to do so.
+Users are grouped into user groups.  
+By default, there are two autogenerated user groups: 
+[the Tenant Administrators](/docs/pe/user-guide/ui/users/#tenant-administrator-user-group) and [the Tenant Users](/docs/pe/user-guide/ui/users/#tenant-user-user-group).
+Tenant administrator can directly create new users in autogenerated user groups, however please note that they will have the access to all dashboards as tenant administrator that has created them.
+Also, there is the "All" user group. If a user is created directly in the "All" group, he won't have any permissions and will view nothing.
+
+{% include images-gallery.html imageCollection="users-intro-pe" showListImageTitles='true' %}
+
+**Note**: outgoing email settings have to be properly configured using [system administrator](/docs/pe/user-guide/ui/tenants/) account. This is required to send activation email to the new users.
+
+## User groups
+
+A User group is group of users of the same level with the same permissions. One user can simultaneously belong to several user groups.
+
+##### Tenant Administrator user group
+
+The created tenant administrators in the Tenant Administrator user group have the same permissions as the tenant administrator who added them to the administrators group (except they can not delete their administrator).
+Let's create a tenant administrator to see what he can do.
+
+{% include images-gallery.html imageCollection="user-bob-pe" showListImageTitles='true' %}
+
+##### Tenant User user group
+
+Tenant users group has read-only permissions. Let's create a tenant user to see what he can do.
+
+{% include images-gallery.html imageCollection="user-alice-pe" showListImageTitles='true' %}
+
+##### Adding a new user group
+
+For the tenant administrator it is possible to create your own user group with customized permissions. 
```

Comment:
Tenant or Customer administrators are able to create user groups with custom permissions.

Reply:
corrected